### PR TITLE
AccountModule: update "provider" copy to move towards wallets and accounts

### LINKED ISSUE #159: AccountModule: update "provider" copy to move towards wallets and accounts
See https://github.com/aragon/court-dashboard/pull/126#discussion_r377702525.

Changes we could do:

- [X]  Replace “Enable account” by “Connect account”.

- [x] Replace “Ethereum providers” by “Use account from”.
- [x] Replace “What is an Ethereum provider” by “Don’t have an Ethereum account?”
- [x] Replace “Active wallet” by “Active account” on the connected state
- [x] Replace "Ethereum providers" by "Connection error" on the error state

### PR DESCRIPTION
**Pull request by Evalir**
Closes #159, #349.

Also solves a bug in the Onboarding Modal: clicking the Close button (The X) doesn't close the modal.

Screenshots:
- Providers before:
Screen Shot 2020-06-03 at 10 23 13 AM
- Providers after (only two providers as I'm running it locally):
Screen Shot 2020-06-03 at 10 22 38 AM

- Connection error before:
Screen Shot 2020-06-03 at 10 22 54 AM
- Connection error after:
Screen Shot 2020-06-03 at 10 22 47 AM

- Connected before:
Screen Shot 2020-06-03 at 10 23 52 AM
- Connected after:
Screen Shot 2020-06-03 at 10 23 29 AM

(Note that it kinda bugs me that we're using "account now" but we have "Disconnect wallet". It's accurate, but I think we could also change this to "Disconnect account")

### CODE REVIEW

File: src/components/Onboarding/OnboardingModal.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ const OnboardingModal = React.memo(function OnboardingModal({
         const compactMode = width < 500 || height < 400
         return (
           <Modal
+            onClose={onComplete}
```
Comment (fabriziovigevani) [inline src/components/Onboarding/OnboardingModal.js:37]: Ah note that the intention here is for the Modal to be not closable see https://github.com/aragon/court-dashboard/issues/349#issuecomment-638025200. We should actually remove the cross icon :D

**Comment by fabriziovigevani** [comment]
> (Note that it kinda bugs me that we're using "account now" but we have "Disconnect wallet". It's accurate, but I think we could also change this to "Disconnect account")

Yeah i agree, i would advocate to for consistency and change every `wallet` to `account`.

**Comment by sohkai** [comment]
> I think we could also change this to "Disconnect account"

Agreed as well; let's change this here and in the client!

**Comment by Evalir** [comment]
I think "Disconnect" can work. I still think that "Disconnect account" looks better, due to the consistency factor.

> I can understand why we used "wallet"—technically the account comes from your wallet, and you can change your account outside of the app via the wallet.

Yes, that's why I say it's accurate wording. Still, if we're shifting away from using "wallet", I don't think it makes sense to keep it for this very particular case.

File: src/components/Onboarding/OnboardingModal.js (hunk 1 above)
Comment (Evalir) [reply src/components/Onboarding/OnboardingModal.js:37]: Ah, cool! Will remove it then, makes sense due to how the `onComplete` function behaves. 👍

File: src/components/Onboarding/Navigation.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ const buttonTransitionStyles = show => ({
   opacity: Number(show),
   transform: `translate3d(0, ${show ? 0 : 2}px, 0)`,
   config: springs.swift,
+  pointerEvents: show ? 'auto' : 'none',
```
Comment (Evalir) [inline src/components/Onboarding/Navigation.js:11]: Another little fix here: The modal buttons have pointer-events even if they are not being shown as only the opacity changes, and this prevents this behavior. The upgrade organization modal from the client also had this, and during spring cleaning [we applied the same fix](https://github.com/aragon/aragon/blob/master/src/components/Upgrade/Navigation.js#L10).

**Comment by sohkai** [comment]
> Still, if we're shifting away from using "wallet"

Ah, the intention was actually to shift towards using "wallet" instead of "provider".

A "wallet" holds a number of "accounts".

**Comment by sohkai** [comment]
> I still think that "Disconnect account" looks better, due to the consistency factor.

Let's punt on this for later, as we work on our nomenclature for specifying ethereum providers / wallets / accounts.

Can we merge this now?

**Comment by rperez89** [comment]
> (Note that it kinda bugs me that we're using "account now" but we have "Disconnect wallet". It's accurate, but I think we could also change this to "Disconnect account")

ahh yes that was my bad, the intention was to start using account everywhere but i forgot about the disconnect

**Comment by Evalir** [comment]
> Can we merge this now?

There is still changes to do left (the height measuring, mainly) which I'll get to very soon as we're finishing up the local connection fixes, but as this PR is mainly concerned about copy aligning, I believe we can merge this and continue on another one.

**Comment by sohkai** [comment]
> as this PR is mainly concerned about copy aligning, I believe we can merge this and continue on another one

Let's merge (since this has fixed the bug with the onboarding modal, and we should've merged that a _long_ time ago 😄), and please raise another issue (and/or the PR) for finishing the updates to the account modal!

**Pull request merged**